packages/keystatic/src/component-blocks/cloud-image-preview.tsx

```diff
@@ -2,159 +2,410 @@
-          <TextField
-            label="Cloud Image"
-            value={props.fields.src.value}
-            onChange={props.fields.src.onChange}
-            onPaste={onPaste}
-            flex="1"
+        <Flex justifyContent="center">
+          <img
+            alt={image.alt}
+            src={image.src}
+            className={css({
+              maxWidth: image.width,
+              maxHeight: image.height,
+            })}
           />
-          {config.storage.kind === 'cloud' &&
-            (() => {
-              const [team, project] = config.storage.project.split('/');
-              return (
-                <div>
-                  <ActionButton
-                    href={`https://keystatic.cloud/teams/${team}/project/${project}/images`}
-                    target="_blank"
-                    rel="noreferrer noopener"
-                  >
-                    <Text>Browse</Text>
-                    <Icon src={externalLinkIcon} />
-                  </ActionButton>
-                </div>
-              );
-            })()}
-          <div>
-            <ActionButton onPress={props.onRemove}>Remove</ActionButton>
-          </div>
         </Flex>
-        <img alt={alt} src={isValidURL(src) ? src : ''} />
-        <Flex gap="regular" alignItems="end">
-          <TextField
-            label="Width"
-            value={props.fields.width.value}
-            onChange={props.fields.width.onChange}
-            width="scale.1200"
-          />
-          <TextField
-            label="Height"
-            value={props.fields.height.value}
-            onChange={props.fields.height.onChange}
-            width="scale.1200"
-          />
-          {dimensions &&
-          (`${dimensions.width}` !== props.fields.width.value ||
-            `${dimensions.height}` !== props.fields.height.value) ? (
-            <div>
-              <ActionButton
-                isDisabled={dimensions === null}
-                alignSelf="start"
-                onPress={() => {
-                  if (dimensions) {
-                    props.onChange({
-                      height: dimensions.height.toString(),
-                      width: dimensions.width.toString(),
-                    });
-                  }
-                }}
+        <Flex padding="large" gap="medium" borderTop="neutral">
+          <VStack flex="1" gap="large" justifyContent="center">
+            <Box>
+              <Text
+                size="small"
+                weight="bold"
+                UNSAFE_className={css({
+                  fontFamily: tokenSchema.typography.fontFamily.code,
+                })}
               >
-                Set to{' '}
-                {dimensions
-                  ? `${dimensions.width} x ${dimensions.height}`
-                  : null}
-              </ActionButton>
-            </div>
-          ) : null}
+                W {image.width} x H {image.height}
+              </Text>
+            </Box>
+            <Box>
```

Comment:
What do these do?

Reply:
Boxes (block) around the Text (inline) components so they get spaced out correctly by the VStack